notebooks/getTimers.sh

```diff
@@ -0,0 +1,46 @@
+if [[ $# < 1 ]]
```

Comment:
We should remove the clear output from the notebook, binary files are no fun in diffs :)